### PR TITLE
[ENH] `dev-docs` for deep learning and zero-shot forecasting

### PR DESCRIPTION
**Pull request by julian-fong**

### CODE REVIEW

**Comment by fkiraly** [comment]
@felipeangelimvieira, FYI

**Comment by julian-fong** [comment]
i already see a few typos still in there but @benHeid @yarnabrina your reviews and feedback would be greatly appreciated!

**Comment by felipeangelimvieira** [comment]
> Since the introduction of the transformer deep learning architecture, the applications of deep learning models has reached new heights, including time series forecasting. Typical time series forecasting involves training a separate model for each time series considered in the dataset. Global forecasting involves creating a single forecaster that simultaneously trains on multiple time series together at once. Since time series datasets are vectorized in nature, global forecasters can be efficiently trained to identify shared patterns between time series

Could this part make the reader think global forecasting only applies to deep learning models? I think the connection to deep learning architectures fits better later, when mentioning pre-trained models and fine-tuning.

Overall, the notebook feels a bit too focused on foundation models — the original idea was something more general about global models, especially reduction forecasters. Still, this works great as a “Deep Learning and Foundation Models” notebook!

Idea: we could assign number 11 for this notebook, for this topic that wasn’t in the original plan, but would be great to have considered. You could open a new issue for this tutorial so we track it separately from the global models one. What do you think?

File: docs/source/tutorials/forecasting/10_global_and_zeroshot_forecasting.ipynb [hunk 1]
```diff
@@ -0,0 +1,1076 @@
+{
```
Comment (benHeid) [inline docs/source/tutorials/forecasting/10_global_and_zeroshot_forecasting.ipynb:1]: <p>What do you mean vectorized? Why is this a requirement? </p><p> </p><p>I would also not say simultaneously. I would just say that is trained on multiple different time series. In fact it sees all time series in the training dataset during training but not all at once in one forward path. </p><p> </p><p>Since this notebook seems to focus on foundation models, It would be sufficient to say that global models are capable of forecasting different time series. </p><p> </p><p>The sentence retaining the strengths of the original foundation model what do you mean with this? I would guess that when you are finetuning a foundation model you will loose some of its general forecasting performance. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime/pull/8910/discussion/#comment-2427135906'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (benHeid) [inline docs/source/tutorials/forecasting/10_global_and_zeroshot_forecasting.ipynb:1]: <p>Why are you using this model? I would argue that this is not really a foundation model since it is only trained on a specific dataset (etth1) or? So it is a global model and a pretrained model but not real foundation model for all types of time series.</p><p> </p><p>I cannot find this model on huggingFace.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime/pull/8910/discussion/#comment-2427135907'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (benHeid) [inline docs/source/tutorials/forecasting/10_global_and_zeroshot_forecasting.ipynb:1]: <p>I would add a note to the model configuration parameter, that changes here might change the architecture of the model. I.e., there might exist newly initialised parameters when changing the configuration. Such a model would probably not perform any useful prediction anymore until it is trained again. </p><p> </p><p>Also the last sentence is not completely true, since also the training arguments determine the ability to learn from data. E.g. if the learning rate is too small (e.g. 0), the model will not learn anything. So I would remove the sentence about the ability to learn from data.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime/pull/8910/discussion/#comment-2427135908'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (benHeid) [inline docs/source/tutorials/forecasting/10_global_and_zeroshot_forecasting.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #23.    forecaster.fit(df[:-192], fh=range(1, 96))  # doctest: +SKIP
</pre><p>Above you mentioned that you are only training parts of the model. So I would add a sentence describing what determines which parts are trained.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime/pull/8910/discussion/#comment-2427135911'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (benHeid) [inline docs/source/tutorials/forecasting/10_global_and_zeroshot_forecasting.ipynb:1]: <p>What are you retraining? Only the embedding or projection layer? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime/pull/8910/discussion/#comment-2427135912'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (benHeid) [inline docs/source/tutorials/forecasting/10_global_and_zeroshot_forecasting.ipynb:1]: <p>I would also add the prediction of model that is not fine-tuned. </p><p> </p><p>I would also suggest that you are using the <code>plot_series</code> method implemented in sktime. This makes the code smaller and also ensures that the plots across the notebooks looks the same. </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime/pull/8910/discussion/#comment-2427135913'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (benHeid) [inline docs/source/tutorials/forecasting/10_global_and_zeroshot_forecasting.ipynb:1]: <p>I think this cell is not executed? There is no plot</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime/pull/8910/discussion/#comment-2427135916'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (benHeid) [inline docs/source/tutorials/forecasting/10_global_and_zeroshot_forecasting.ipynb:1]: <p>I would propose to use <code>temporal_train_test_split</code> from sktime.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime/pull/8910/discussion/#comment-2427135918'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (benHeid) [inline docs/source/tutorials/forecasting/10_global_and_zeroshot_forecasting.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #6.    if isinstance(y_test, pd.DataFrame):
</pre><p>Why do you need this distinction here? I would guess only one of the cases should be executed? </p><p> </p><p>I would also recommend again to use <code>plot_series</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime/pull/8910/discussion/#comment-2427135920'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

**Comment by felipeangelimvieira** [comment]
Hello @julian-fong ! Are you planning to continue this PR with the provided feedback?

**Comment by fkiraly** [comment]
the design will probably change a little given @SimonBlanke´s PR https://github.com/sktime/sktime/pull/9056

**Comment by julian-fong** [comment]
Hi @felipeangelimvieira i don't plan on working on this anytime soon, i will close the pr! thanks
